.github/workflows/esti.yaml

```diff
@@ -498,13 +499,29 @@ jobs:
         working-directory: test/spark
         run: |
           python ./run-test.py \
-            --storage_namespace s3://esti-system-testing/${{ github.run_number }}-spark${{ matrix.spark.tag }}-client/${{ steps.unique.outputs.value }} \
-            --repository thick-client-test \
+            --storage_namespace s3://esti-system-testing/${{ github.run_number }}-spark${{ matrix.spark.tag }}-lakefsfs/${{ steps.unique.outputs.value }} \
+            --repository lakefsfs-test-spark \
             --sonnet_jar ${{ matrix.spark.sonnet_jar }} \
             --access_mode hadoopfs \
             --aws_access_key ${{ secrets.ESTI_AWS_ACCESS_KEY_ID }} \
             --aws_secret_key ${{ secrets.ESTI_AWS_SECRET_ACCESS_KEY }}
       - name: lakeFS Logs on HadoopFS test failure
+        if: ${{ failure() }}
+        continue-on-error: true
+        working-directory: test/spark
+        run: docker-compose logs --tail=2500 lakefs      
+
+      - name: Spark${{ matrix.spark.tag }} + lakeFS GW + Redirect
+        if: env.SPARK_TAG == '3'
```

Comment:
Why?

Reply:
Because this does not work with Spark2.
Also introducing new functionality we should step back from Spark2 as a rule